definitions/npm/cookie_v0.3.x/flow_v0.58.x-/cookie_v0.3.x.js

```diff
@@ -0,0 +1,22 @@
+type SerializeOptions = {
+  domain?: ?string,
+  encode?: ?((stringToDecode: string) => string),
+  expires?: ?Date,
+  httpOnly?: ?boolean,
+  maxAge?: ?number,
+  path?: ?string,
+  sameSite?: ?(boolean | 'lax' | 'strict'),
+  secure?: ?boolean,
```

Comment:
The double question mark is unnecessary. They're optional props, but I'm assuming they're not maybe types, so it should be `domain?: string` for example.

Reply:
Library itself accept both options. There are checks:
```
if (options.path) {
// do something
}
```

Should I change it to ```domain?: string``` or leave it as it is?